doc/tutorials/pre-launched-rt.rst

```diff
@@ -30,60 +30,60 @@ Example of Pre-Launched RT
 
 Take the Whiskey Lake WHL-IPC-I5 board (as described in :ref:`hardware`) for
 example. This platform can connect both an NVMe and a SATA drive and has
-two Ethernet ports. We will passthrough the SATA and Ethernet 03:00.0
-devices into the Pre-Launched RT VM, and give the rest of the devices to
+two Ethernet ports. We will pass through the SATA and Ethernet 03:00.0
```

Comment:
Nearly every use in our documentation is spelled "passthrough" (211 uses).

Reply:
> Nearly every use in our documentation is spelled "passthrough" (211 uses).

In general, "pass through" is 2 words when it's a verb: passes through, pass through, passing through, passed through. But when it's an adjective, it's "passthrough." Similar to the words "set up" (verb) and "setup" (adjective/noun). Otherwise, it's awkward when you need to conjugate the verb form.